esp/platform/espcontext.cpp

```diff
@@ -466,9 +466,6 @@ class CEspContext : public CInterface, implements IEspContext
     {
         if (m_txSummary)
         {
-            if (!m_hasException && (getEspLogLevel() <= LogNormal))
```

Comment:
Agreed, we had to do this in order to get reasonable tx summaries for the stress test they performed... but now I'm wondering if we should/need to output the extended txsummary for all loglevels, I think we should NOT output the txsummary details if loglevel = 1 (production). Anything higher than that we can output the details.
Also, perhaps outside of the scope of this issue, but I wish we could suppress txsummaries for front-end generated requests... GET File x GET JS, etc....